### PR TITLE
Add description for assignment0

### PR DESCRIPTION
**Pull request by zhengp0**
* Add description for assignment 0

### CODE REVIEW

File: assignment0/description.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+In this first assignment, we will set up the GitHub repository needed for all
+the following assignments. Please follow the steps below to create the
+repository and submit a Pull Request (PR) for Assignment 0.
+
+## Create Assignment Repository
+
+* Create a GitHub account if you do not have one, preferably using your uw
+email.
+* Create a **private** repository on GitHub with name `HMS520-Autumn2022`.
+* Give Adrien `jiaweih` and me `zhengp0` access to the repository with read
```
Comment (jiaweih) [inline assignment0/description.md:12]: Not Adrien any more😄. Do we also need to put full email address here? Or account name is enough?

File: assignment0/description.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Structure
+In this repository, we will use folders with name `assignment{i}` to host the
+submission of each assignment, where `i` is homework id.
+If there are useful tips and insights for programming in R, they shall go to the
+`notes` folder.
+```
+
+* Submit a PR from `assignment0` branch to `main` branch, with title
+`Assignment0` and `Describe structure of the repository.` as the description.
+* Add Adrien `jiaweih` and me `zhengp0` as the reviewers.
```
Comment (jiaweih) [inline assignment0/description.md:33]: Similarly here, not Adrien any more.

File: assignment0/description.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Assignment 0
+
+In this first assignment, we will set up the GitHub repository needed for all
+the following assignments. Please follow the steps below to create the
+repository and submit a Pull Request (PR) for Assignment 0.
+
+## Create Assignment Repository
```
Comment (jiaweih) [inline assignment0/description.md:7]: We don't have to do it here, but just so you should know that another way to do this is through Github Classroom.

File: assignment0/description.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* Create a GitHub account if you do not have one, preferably using your uw
+email.
+* Create a **private** repository on GitHub with name `HMS520-Autumn2022`.
+* Give Adrien `jiaweih` and me `zhengp0` access to the repository with read
+  and write permission.
+* Either link your local repository or start fresh, commit a `README.md` file to
```
Comment (jiaweih) [inline assignment0/description.md:14]: I understand this step, but I'm wondering if the students would understand.

File: assignment0/description.md (hunk 1 above)
Comment (zhengp0) [reply assignment0/description.md]: Thanks @jiaweih! Sorry forgot to change that. I think the Github account here is alright :) Let me know if I got your Github account wrong.

File: assignment0/description.md (hunk 3 above)
Comment (zhengp0) [reply assignment0/description.md:7]: Thanks Jiawei, I actually prefer students to go through the process to create their own repo :)

File: assignment0/description.md (hunk 4 above)
Comment (zhengp0) [reply assignment0/description.md:14]: Yes! We will walk it through during the class.

**Review by jiaweih — APPROVED** [review]
Looks good!

File: assignment0/description.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Assignment 0
```
Comment (jiaweih) [inline assignment0/description.md:1]: For students' questions regarding the assignment, I was thinking of two ways, either ask them to discuss on Canvas or discuss with the Issues on this repository. What do you think?

Comment (zhengp0) [reply assignment0/description.md:1]: I think canvas is the better choice, so all the info is in one place :)

Comment (jiaweih) [reply assignment0/description.md:1]: Sure

**Pull request merged**